libpoolprotocols/stratum/EthStratumClient.cpp

```diff
@@ -277,11 +277,11 @@ void EthStratumClient::disconnect_finalize() {
 }
 
 void EthStratumClient::resolve_handler(const boost::system::error_code& ec, tcp::resolver::iterator i)
-{	
+{
+    dev::setThreadName("stratum");
```

Comment:
Please leave setThreadName only immediately before cwarn.
No need to force the name if we have nothing to write in output.